simplified-ui-theme/src/main/res/values/styles.xml

```diff
@@ -108,6 +108,12 @@
     <item name="android:background">@drawable/simplified_button</item>
     <item name="android:textColor">@drawable/simplified_button_text</item>
     <item name="android:textSize">12sp</item>
+
+    <!--
+      The scroll indicator will briefly appear on programmatically created buttons unless the
+      'scrollbars' attribute is set to 'none'.
+    -->
+    <item name="android:scrollbars">none</item>
```

Comment:
Turns out when we add buttons dynamically Android somehow thinks they're scrollable. I have no idea why this is happening. I did some searching and read through some of the `View` source code but I couldn't figure out exactly what was happening. However, adding this attribute resolves the issue and I can't imagine why a button would need to be internally scrollable anyway.